nflow-rest-api/src/main/java/com/nitorcreations/nflow/rest/v1/converter/ListWorkflowInstanceConverter.java

```diff
@@ -83,8 +82,8 @@ private JsonNode stringToJson(String key, String value) {
     try {
       return nflowRestObjectMapper.readTree(value);
     } catch (IOException e) {
-      logger.warn("Failed to parse state variable {} value as JSON, returning value as unparsed string: {}: {}",
-    		  key, e.getClass().getSimpleName(), e.getMessage());
+      logger.debug("Failed to parse state variable {} value as JSON, returning value as unparsed string: {}: {}", key,
```

Comment:
Yes it is needed, altough the whole json conversion is controversial. State variable values are not necessarily always json.